pkg/mgrconfig/load.go

```diff
@@ -86,14 +86,15 @@ func LoadPartialFile(filename string) (*Config, error) {
 
 func defaultValues() *Config {
 	return &Config{
-		SSHUser:        "root",
-		Cover:          true,
-		Reproduce:      true,
-		Sandbox:        "none",
-		RPC:            ":0",
-		MaxCrashLogs:   100,
-		Procs:          6,
-		PreserveCorpus: true,
+		SSHUser:           "root",
+		Cover:             true,
+		Reproduce:         true,
+		Sandbox:           "none",
+		RPC:               ":0",
+		MaxCrashLogs:      100,
+		Procs:             6,
+		PreserveCorpus:    true,
+		StraceBinOnTarget: false,
```

Comment:
I would remove this. We try not to explicitly spell default values (usually it's the right default value).